languages/javascript/exercises/concept/booleans/booleans.js

```diff
@@ -1,15 +1,45 @@
+/**
+ * If the knight is sleeping, then the fast attack is not available.
+ *
+ * @param {boolean} knightIsAwake
+ * 
+ * @return {boolean} Whether or not you can execute a fast attack.
+ */
 export function canExecuteFastAttack(knightIsAwake) {
   throw new Error('Implement the canExecuteFastAttack function')
 }
 
+/**
+ * You can only spy if someone is awake; you wouldn't want to go through all the trouble just to find out nothing's going to happen, would you?
```

Comment:
Can you limit these lines to 80 characters? It will look better on smaller screens.